cmd/server/main.go

```diff
@@ -76,12 +81,35 @@ func main() {
 		klog.Exitf("--config is required")
 	}
 
+	if *githubAPIRawURL != "" && *githubAPIRawUploadURL == "" {
+		klog.Exitf("--github-api-upload-url must be set when -github-api-base-url was set")
+	}
+
 	ctx := context.Background()
 
 	client := github.NewClient(oauth2.NewClient(ctx, oauth2.StaticTokenSource(
 		&oauth2.Token{AccessToken: triage.MustReadToken(*githubTokenFile, "GITHUB_TOKEN")},
 	)))
 
+	// set GitHub API endpoints to client if custom endpoint was set.
```

Comment:
The same logic should also be applied to `cmd/tester/main.go`.

Do you mind moving this code to `pkg/triage/github.go`? This is where I've been sneaking the logic that is shared between the two programs.

Reply:
Thanks for the catch. I will do it.